### PR TITLE
Check sell and update sql trade table.

### PR DESCRIPTION
**Pull request by creslinux**
This is to sync trade table after selling.

Currently this is only done after a full loop of all pairs, which can
take over 2+ minutes if 100 are loaded. See #1053

Currently when a few trades are closed we will wait minutes plus, then when 1 record on trades table is updated the others are also updated in the same second, giving the appearance  that all trades exited in the same, but late, moment. 

This is problematic as the open trades count is false, and any tooling checking for open trades awry, later reporting has the wrong times recorded. 

Originally submitted as a call to maybe_sell def, which should never have created a loop as only called after an order_id is set - but to be doubly safe have split out the discreet logic interested in.

### CODE REVIEW

**Comment by creslinux** [comment]
video side/by/side trades table updating / not in correct time. 
without change open trades are marked closed in mass when all/last open trades is marked closed.

File: freqtrade/freqtradebot.py [hunk 1]
```diff
@@ -619,6 +619,24 @@ def execute_sell(self, trade: Trade, limit: float) -> None:
...
+            # Update trade with order values
+            logger.info('Found open order for %s', trade)
+            order = self.exchange.get_order(trade.open_order_id, trade.pair)
+            # Try update amount (binance-fix)
+            try:
+                new_amount = self.get_real_amount(trade, order)
+                if order['amount'] != new_amount:
+                    order['amount'] = new_amount
+                    # Fee was applied, so set to 0
+                    trade.fee_open = 0
```
Comment (xmatthias) [inline freqtrade/freqtradebot.py]: i think during sell you should set fee_close, not fee_open.

**Comment by creslinux** [comment]
Hi the call is not a fraction of a second later, in tests it's 1-2 seconds later. 

It is prudent to check the sell has completed 1-2 seconds after being placed than waiting an unknown period of time, the main loop can take minutes.

File: freqtrade/freqtradebot.py (hunk 1 above)
Comment (creslinux) [reply freqtrade/freqtradebot.py]: Good spot, ill have to check source but that may be a bug in original code I reused! 

These is the chance I reused the wrong section, this PR took 5 attempts before my GIT stopping being one.

**Comment by xmatthias** [comment]
Going to close this for inactivity...

**Pull request closed without merging**